applications/GeoMechanicsApplication/tests/cpp_tests/test_GeoMechanicsNewtonRaphsonErosionProcessStrategy.cpp

```diff
@@ -26,10 +26,17 @@
 namespace Kratos::Testing
 {
 
+std::string WorkingDirectorySellmeijer()
+{
+    auto projectDirectory = "./applications/GeoMechanicsApplication/tests/test_compare_sellmeijer/HeightAquiferD10L30.gid";
+    auto kratos_source_directory = std::getenv("KRATOS_SOURCE");
+    return (std::string)kratos_source_directory + "/" + (std::string)projectDirectory;
```

Comment:
I strongly recommend to use `std::filesystem::path` for anything related to file paths

Reply:
Thank you for pointing. string is replaced with path.